apps/next/features/banner/bannerRefresh.ts

```diff
@@ -0,0 +1,53 @@
+import prisma from "@app/util/ssr/prisma";
+import { Feature } from "@app/features/Feature";
+import { flipFeatureEnabled, getTwitterInfo } from "@app/util/database/postgresHelpers";
+import { getAccountsById } from "@app/util/getAccountsById";
+import { logger } from "@app/util/logger";
+import { TwitterResponseCode, updateBanner, validateTwitterAuthentication } from "@app/util/twitter/twitterHelpers";
+import { renderBanner } from "./renderBanner";
+
+export const bannerRefresh: Feature<string> = async (userId: string): Promise<string> => {
+    /**
+     * 1. Render banner image
+     * 2. Check if user is still live according to our database
```

Comment:
We may still have some users from the first test run that are not actually live but are in the live db. We also may hit something where they are accidently kept in the live user table. We should handle this properly, if we are talking with twitch and don't get anything back just void the change.

Reply:
Ok I made it return if they aren't live from the twitch api